### ISSUE TITLE
OTRS Migration fails with Encoding Error

### ISSUE THREAD

**Issue by Floaty-Pi** [zammad/zammad#3385]
### Infos:

* Used Zammad version: 3.5.03
* Installation method: Univention (Version 4.4-7)
* Operating system: CentOS 7.8.2003
* Database + version: mySQL 15.1

### Expected behavior:

* Migration completes successfully 

### Actual behavior:

* Migration runs smoothly until it terminates at ~13k tickets and stops with an EncriptionError 
```
I, [2021-01-25T13:07:41.826777 #38-47219306277240]  INFO -- : Won't enqueue SearchIndexJob (Job ID: 5da6a60e-713a-4ea9-933d-10a44dd7bc97) because of already existing job with lock key 'SearchIndexJob/Ticket/13332'.
I, [2021-01-25T13:07:42.026339 #38-47219306277240]  INFO -- : Enqueued SearchIndexJob (Job ID: 5da6a60e-713a-4ea9-933d-10a44dd7bc97) to DelayedJob(default) with arguments: "Ticket", 13332
I, [2021-01-25T13:07:43.034557 #38-47219306277240]  INFO -- : Enqueued SearchIndexAssociationsJob (Job ID: f62eb886-c9ac-4954-9775-a1e4ca212f36) to DelayedJob(default) with arguments: "Ticket", 13332
#<Thread:0x000055e433f44cd8@/usr/local/bundle/gems/logging-2.2.2/lib/logging/diagnostic_context.rb:471 run> terminated with exception (report_on_exception is true):
Traceback (most recent call last):
        12: from /usr/local/bundle/gems/logging-2.2.2/lib/logging/diagnostic_context.rb:474:in `block in create_with_logging_context'
        11: from /opt/zammad/lib/import/otrs.rb:93:in `block (2 levels) in threaded_import'
        10: from /opt/zammad/lib/import/otrs.rb:93:in `loop'
         9: from /opt/zammad/lib/import/otrs.rb:99:in `block (3 levels) in threaded_import'
         8: from /opt/zammad/lib/import/otrs.rb:135:in `imported?'
         7: from /opt/zammad/lib/import/otrs.rb:141:in `import_action'
         6: from /opt/zammad/lib/import/otrs/requester.rb:32:in `load'
         5: from /opt/zammad/lib/import/otrs/requester.rb:77:in `request_result'
         4: from /opt/zammad/lib/import/otrs/requester.rb:91:in `request_json'
         3: from /opt/zammad/lib/import/otrs/requester.rb:98:in `handle_response'
         2: from /opt/zammad/lib/core_ext/object.rb:3:in `to_utf8'
         1: from /opt/zammad/lib/core_ext/string.rb:477:in `utf8_encode'
/opt/zammad/lib/core_ext/string.rb:511:in `utf8_encode!': could not find a valid input encoding (EncodingError)
Traceback (most recent call last):
        12: from /usr/local/bundle/gems/logging-2.2.2/lib/logging/diagnostic_context.rb:474:in `block in create_with_logging_context'
        11: from /opt/zammad/lib/import/otrs.rb:93:in `block (2 levels) in threaded_import'
        10: from /opt/zammad/lib/import/otrs.rb:93:in `loop'
         9: from /opt/zammad/lib/import/otrs.rb:99:in `block (3 levels) in threaded_import'
         8: from /opt/zammad/lib/import/otrs.rb:135:in `imported?'
         7: from /opt/zammad/lib/import/otrs.rb:141:in `import_action'
         6: from /opt/zammad/lib/import/otrs/requester.rb:32:in `load'
         5: from /opt/zammad/lib/import/otrs/requester.rb:77:in `request_result'
         4: from /opt/zammad/lib/import/otrs/requester.rb:91:in `request_json'
         3: from /opt/zammad/lib/import/otrs/requester.rb:98:in `handle_response'
         2: from /opt/zammad/lib/core_ext/object.rb:3:in `to_utf8'
         1: from /opt/zammad/lib/core_ext/string.rb:477:in `utf8_encode'
/opt/zammad/lib/core_ext/string.rb:511:in `utf8_encode!': could not find a valid input encoding (EncodingError)
```

### Steps to reproduce the behavior:

* open ruby console
* extend the runtime to 7 days  
* enter link and key provided by the migration plug-in
* start migration 

Yes I'm sure this is a bug and no feature request or a general question.

**Comment by martini** [comment]
@Floaty-Pi it seems to be a duplicate of #3250 which git fixed in Zammad 3.6 and higher.

Pls. verify if it's working with Zammad 3.6. Thanks!

**Comment by MrGeneration** [comment]
Zammad 3.6 is not available on Univention systems as of now.
I'm afraid testing this is not possible as of now.

No ETA for this release before you ask.
